src/dlgTriggerEditor.cpp

```diff
@@ -6957,25 +6957,27 @@ void dlgTriggerEditor::slot_script_main_area_delete_handler()
 
 void dlgTriggerEditor::slot_script_main_area_add_handler()
 {
+    auto createItemList = [&] () {
+        auto pItem = new QListWidgetItem;
+        pItem->setText(mpScriptsMainArea->lineEdit_script_event_handler_entry->text());
+        mpScriptsMainArea->listWidget_script_registered_event_handlers->addItem(pItem);
+    };
+
     mpScriptsMainArea->trimEventHandlerName();
     if (mIsScriptsMainAreaEditHandler) {
```

Comment:
Just a comment for the future, not this PR, we cannot have such confusing variable names. "if is scripts main area edit handler" makes no sense, and it takes too long to understand what it does!

Reply:
well i'm not responsible of this very long_long_long_variable_name. hehe.

Reply:
Long isn't the problem, the problem is that it doesn't make any sense. Long and meaningful is totally cool. `if (editingEventHandler)` for example would make complete sense right away.

Reply:
then maybe in other PR we can refactor this.